appcenter-crashes/android/src/main/java/com/microsoft/appcenter/reactnative/crashes/AppCenterReactNativeCrashesUtils.java

```diff
@@ -41,13 +41,11 @@ static WritableMap convertErrorReportToWritableMap(ErrorReport errorReport) thro
         errorReportMap.putString("threadName", errorReport.getThreadName());
         errorReportMap.putString("appErrorTime", "" + errorReport.getAppErrorTime().getTime());
         errorReportMap.putString("appStartTime", "" + errorReport.getAppStartTime().getTime());
-        
-        //noinspection ThrowableResultOfMethodCallIgnored
-        Throwable error = errorReport.getThrowable();
-        if (error != null) {
-            errorReportMap.putString("exception", Log.getStackTraceString(error));
-            errorReportMap.putString("exceptionReason", error.getMessage());
+        String stackTrace = errorReport.getStackTrace();
+        if (stackTrace != null) {
+            errorReportMap.putString("exception", stackTrace);
```

Comment:
Wouldn't this change cause a test to fail?  Do any tests need to be added?

Reply:
I think either way you are just putting a string in to "exception" of the errorReportMap. Not sure if there is already a test for this, but seems likely it would continue to pass with this change

Reply:
Well, we know the code is being covered, but my suspicion is that the contents of the entry in the map are not checked

Reply:
@jwallra @michaelsmitha We have no unit tests for RN Java or Objective-C bridge code (only tests for JavaScript). Filling the test gap is something we want to do but never got to it.